include/sys/arc_impl.h

```diff
@@ -240,6 +240,9 @@ struct arc_buf_hdr {
 	uint64_t		b_birth;
 
 	arc_buf_contents_t	b_type;
+	uint8_t			b_complevel;
+	uint8_t			b_reserved1; /* used for 4 byte alignment */
+	uint16_t		b_reserved2; /* used for 4 byte alignment */
```

Comment:
Just info:
We had a discussion about this, but it is actually a previous review request from Behlendorf:
https://github.com/zfsonlinux/zfs/pull/8044#discussion_r227119700

Reply:
no sure if you want to change a thing here

Reply:
No I dont, but we had a discussion about it on brainslayers branch, just being thorough by adding this was added this was for a reason, so it should stay.

Reply:
ok, thank you